src/action.ts

```diff
@@ -8,37 +8,44 @@
 
 'use strict';
 
-const utils = require('./utils');
+import { Device } from './device';
+import { timestamp } from './utils';
+
+export interface ActionDescription {
+  id: string
+  name: string
+  input: any,
```

Comment:
Basing on `tsconfig.json → noImplicitAny: true` compiler should raise error for any `any` (pun intended)?

Reply:
That's actually an explicit `any` :grimacing:

Reply:
Indeed :-D Apologies for confusion!